src/core/jit/context.rs

```diff
@@ -40,10 +41,39 @@ impl<'a> ResolverContextLike for Context<'a, async_graphql::Value, async_graphql
     }
 
     fn is_query(&self) -> bool {
-        todo!()
+        self.request
+            .document
+            .operations
+            .iter()
+            .any(|(_, op)| op.node.ty == OperationType::Query)
```

Comment:
in theory document can contain multiple operations where some of them could be mutations, but here we'll mark all of them as query. 

I think `is_query` should be defined specific for the currently processed operation